test/integration/backward_compatible/parallel/serialization/compact/CompactTest.js

```diff
@@ -78,7 +78,8 @@ describe('CompactTest', function () {
     before(async function () {
         TestUtil.markClientVersionAtLeast(this, '5.1.0');
         const comparisonValueForServerVersion520 = await TestUtil.compareServerVersionWithRC(RC, '5.2.0');
-        if ((await TestUtil.compareServerVersionWithRC(RC, '5.1.0')) < 0) {
+        // Compact serialization 5.2 and newer clients are not compatible with servers older than 5.2
+        if (comparisonValueForServerVersion520 < 0 && TestUtil.isClientVersionAtLeast('5.2.0')) {
```

Comment:
I think we should assert client and server should be >= 5.2.0, otherwise skip

Reply:
That is what I had in my mind, which would simplify a lot of things. The only downside is that we won't be able to test 5.1.z server with a 5.1.z client regarding Compact serialization when releasing a new version of either of those. I believe it is totally OK, as the Compact was BETA in those versions anyways

Reply:
Then, we can assert either they are both above or equal 5.2.0, or older than 5.2.0.